qa/rpc-tests/run_until_fails.py

```diff
@@ -0,0 +1,28 @@
+import os.path
+import subprocess
+import sys
+
+from os import path
+
+if len(sys.argv) != 2:
+    print("Usage: run_until_fails.py test_name.py")
```

Comment:
It might be useful adding a comment with the purpose of this script

Reply:
Comment added.